modules/test/playwright/pages/commerce/commerceAdminChannelDetailsPage.ts

```diff
@@ -8,13 +8,22 @@ import {Locator, Page} from '@playwright/test';
 import {ApplicationsMenuPage} from '../product-navigation-applications-menu/ApplicationsMenuPage';
 export class CommerceAdminChannelDetailsPage {
 	readonly applicationsMenuPage: ApplicationsMenuPage;
+	readonly channelNameLink: (channelName: string ) => Promise<Locator>;
+	readonly checkChannelName: (channelName: string ) => Promise<Locator>;
 	readonly countryTab: Locator;
 	readonly page: Page;
 	readonly saveButton: Locator;
 	readonly showSeparateOrderItemsToggle: Locator;
 
 	constructor(page: Page) {
 		this.applicationsMenuPage = new ApplicationsMenuPage(page);
+		this.channelNameLink = async(channelName) => page.getByRole('link', {
+			exact: true,
+			name: channelName,
+		});
+		this.checkChannelName = async(channelName) => {
+			return page.getByText(channelName);
+		};
```

Comment:
you don't need the async because you're not returning a promise but directly the locator